src/resolvers/__tests__/collection.test.js

```diff
@@ -277,7 +277,8 @@ describe('Collection', () => {
             meta: {
               'concept-id': 'C100000-EDSC',
               'native-id': 'test-guid',
-              'revision-id': '2'
+              'revision-id': '3',
```

Comment:
I think these changes to the mocks should be removed unless you plan to add a specific test to do something with the "deleted" mock you added.

Reply:
The deleted mock shows that the whole call doesn't break when umm does not exist. Without adding these line 337 isn't fully tested

Reply:
Agree, suggest including a specific test that utilizes this new mock dtaa, just a simple one like, "when CMR response includes deleted records" "it  includes these records in the response" -- especially since this was a bug that caused a breakage.

Reply:
@cgokey is right, we should leave this test as is, and write a new test to cover this use case